src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      width: "100%",
```

Comment:
Please ensure the item's width span 100% of root width (from lines 19,28 and 53).

Reply:
If the item's width span 100% of the root width, the end of the button will be close to the window. (first picture)
So I design to make the width at root width: "95%" and the item's width span 100% of it. It's looking better. (second picture)
or I have the other way, Can you advise me><
![Screenshot (7)](https://user-images.githubusercontent.com/59549784/126099389-0644630b-be43-4f06-a956-2b506d629ce8.png)
![Screenshot (8)](https://user-images.githubusercontent.com/59549784/126099664-dd1592f5-58c9-47e6-ba63-cfabc2d8e06b.png)

Reply:
Ahh, I see. I forgot to set `box-sizing` property. Therefore, please set property at index.css with below code. After you set that property, all of your components will be **bad**. So, you can fix it by delete all height property and set width with proper value(summation 100%).

```
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: border-box;
}
```

Reply:
And your description looks so good. I appreciate it.